hydrolysis-analyzer.html

```diff
@@ -35,6 +35,14 @@
       },
 
       /**
+       * A list of sources to load in addition to src when analyzing.
+       */
+      sources: {
```

Comment:
seems a little odd to load src + sources. Can `src` just take a `string | string[]`?